test/unit/android-controller-specs.js

```diff
@@ -3,11 +3,16 @@
 var chai = require('chai')
   , controller_path = '../../lib/devices/android/android-controller.js'
   , controller = require(controller_path)
-  , _ = require('underscore');
+  , _ = require('underscore')
+  , sinon = require('sinon');
 
 chai.should();
 
 describe('android-controller', function () {
+  beforeEach(function() {
+    controller.adb = { getApiLevel: function() {} };
```

Comment:
This was a little wonky, this was my first time playing with sinon.  In the spec adb starts out null on the controller.  We needed to define it here for the stub below to work.  Having taken a second look, I would suggest we update this to be: 

`
    controller.adb = {
      getApiLevel: function(cb) {
        cb();
      }
    };
`

And then actually remove the stubbing below.  It isn't really needed since we're not doing any assertions on it anyways and there isn't an original method to restore.  If you agree I'll go ahead and make that change.